gneiss/plot/_regression_plot.py

```diff
@@ -0,0 +1,369 @@
+            fill_color='color', line_color="white", source=source)
+    Xlabels = pd.Series(pvals.index, index=np.arange(len(pvals.index)))
+    Ylabels = pd.Series(pvals.columns, index=np.arange(len(pvals.columns)), )
+
+    hm.xaxis[0].ticker = FixedTicker(ticks=Xlabels.index)
+    hm.xaxis.formatter = FuncTickFormatter(code="""
+    var labels = %s;
+    return labels[tick];
+    """ % Xlabels.to_dict())
+
+    hm.yaxis[0].ticker = FixedTicker(ticks=Ylabels.index)
+    hm.yaxis.formatter = FuncTickFormatter(code="""
+    var labels = %s;
+    return labels[tick];
+    """ % Ylabels.to_dict())
+
+    return hm
+
+
+def _decorate_tree(t, series):
+    """ Attaches some default values on the tree for plotting.
+
+    Parameters
+    ----------
+    t: skbio.TreeNode
+        Input tree
+    series: pd.Series
+        Input pandas series
+
+    """
+    for i, n in enumerate(t.postorder()):
+        n.size = 30
+        if n.is_root():
+            n.size = 50
+        elif n.name == n.parent.children[0].name:
+            n.color = '#00FF00'  # left child is green
+        else:
+            n.color = '#FF0000'  # right child is red
+        if not n.is_tip():
+            t.length = series.loc[n.name]
+    return t
+
+
+def _deposit_results(model, output_dir):
+    """ Store all of the core regression results into a folder. """
+    coefficients = model.coefficients()
+    coefficients.to_csv(os.path.join(output_dir, 'coefficients.csv'),
+                        header=True, index=True)
+    residuals = model.residuals()
+    residuals.to_csv(os.path.join(output_dir, 'residuals.csv'),
+                     header=True, index=True)
+    predicted = model.predict()
+    predicted.to_csv(os.path.join(output_dir, 'predicted.csv'),
+                     header=True, index=True)
+    pvalues = model.pvalues
+    pvalues.to_csv(os.path.join(output_dir, 'pvalues.csv'),
+                   header=True, index=True)
+    balances = model.balances
+    balances.to_csv(os.path.join(output_dir, 'balances.csv'),
+                    header=True, index=True)
+
+
+def _deposit_results_html(index_f):
+    """ Create links to all of the regression results.
+    Parameters
+    ----------
+    index_f : filehandle
+        File handle for dumping the results.
+    """
+    index_f.write(
+        ('<th>Coefficients</th>\n'
+         '<a href="coefficients.csv">'
+         'Download as CSV</a><br>\n'
+         '<th>Coefficient pvalues</th>\n'
+         '<a href="pvalues.csv">'
+         'Download as CSV</a><br>\n'
+         '<th>Raw Balances</th>\n'
+         '<a href="balances.csv.csv">'
+         'Download as CSV</a><br>\n'
+         '<th>Predicted Balances</th>\n'
+         '<a href="predicted.csv">'
+         'Download as CSV</a><br>\n'
+         '<th>Residuals</th>\n'
+         '<a href="residuals.csv">'
+         'Download as CSV</a><br>\n'
+         '<th>Tree</th>\n')
+    )
+
+
+# OLS summary
+def ols_summary(output_dir: str, model: OLSModel,
```

Comment:
This was originally done to fit the q2 standards.  Now it is just another helper function.